gateway/ln-gateway/src/lib.rs

```diff
@@ -118,6 +118,9 @@ impl LnGateway {
             .load_federation_actors(decoders, module_gens, route_hints)
             .await;
 
+        let version = env!("GIT_HASH").to_string();
+        info!("Starting lightning gateway (version: {version}");
```

Comment:
BTW. Just looking at it now, we should use structural logging like this: ` info!(version, "Starting lightning gateway");` (or `%version`).